scripts/install-worker.sh

```diff
@@ -373,4 +373,5 @@ echo vm.max_map_count=524288 | sudo tee -a /etc/sysctl.conf
 sudo mkdir -p /etc/eks/log-collector-script/
 sudo cp $TEMPLATE_DIR/log-collector-script/eks-log-collector.sh /etc/eks/log-collector-script/
 
+sudo rm -f /etc/machine-id 
 sudo touch /etc/machine-id
```

Comment:
The rationale was to allow for `cleanup` to be skipped but still create the image.  I'm fine if we don't ever want clean-up to be skipped with an image creation, since you could still change packer to abort on error rather than terminate on error for debugging.

Reply:
Yeah I'm fine with not supporting skipping cleanup; `validate.sh` will fail if it's not done, and the validation should *always* be done. Moving the cleanup to a separate provisioner was intended to add flexibility on *when* it was done, not whether it was done.

Reply:
maybe add a check for `machine-id` in `validate.sh`?

Reply:
updated